### ISSUE TITLE
ignore.map is not a function

### ISSUE THREAD

**Issue by whyboris** [rtfpessoa/diff2html-cli#100]
Currently using the `--ig` flag does not work:
```
diff2html --ig package-lock.json
```
results in:
```
TypeError: ignore.map is not a function
```
https://github.com/rtfpessoa/diff2html-cli/blob/1c9b955fe8b1c569499feb248ea81b7f749be154/src/cli.ts#L17

I've not looked into it, just reporting 😅

**Comment by whyboris** [comment]
🎉 Looked into it, it's just a matter of updating the `yargs` type 😉 

```
type: 'array',
```
instead of `nargs: 1,` this line:
https://github.com/rtfpessoa/diff2html-cli/blob/master/src/yargs.ts#L177

**Comment by rtfpessoa** [comment]
Released as version `5.1.1`
